odoo/modules/loading.py

```diff
@@ -583,6 +583,7 @@ def load_modules(db, force_demo=False, status=None, update_module=False):
         registry.updated_modules += processed_modules
 
         # OpenUpgrade: run deferred tests
+        cr.commit()
```

Comment:
Without this commit, the transaction with the end stage migration script is rolled back. Probably due to the test run below.